### PR TITLE
chore: enable turbo cache on github workflows

### PR DESCRIPTION
**Pull request by elbalexandre**
## Description

- enable turbo cache on github workflows

## Additional Notes

N/a

## Checklist:

Before you create this PR, confirm all the requirements listed below by checking the checkboxes like this (`[x]`).

- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests to cover the new feature or fixes.

### CODE REVIEW

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -30,6 +30,15 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Cache TurboRepo
```
Comment (ValentinH) [inline .github/workflows/quality.yml:33]: What's the goal for doing this?
It doesn't seem to be Turborepo specific but instead just caching node modules. In that case, this should be done before the "Instal deps" step to really be useful.

If you want to use Turbo remote cache, we should instead setup the right env variables to use our Vercel remote cache plan. See https://turbo.build/repo/docs/ci/github-actions

File: .github/workflows/quality.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ jobs:
     if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ vars.TURBO_TEAM }}
```
Comment (ValentinH) [inline .github/workflows/quality.yml:19]: where is `vars.TURBO_TEAM` defined?

Comment (elbalexandre) [reply .github/workflows/quality.yml:19]: In Github settings

**Review by ValentinH — APPROVED** [review]

File: .github/workflows/quality.yml (hunk 2 above)
Comment (ValentinH) [reply .github/workflows/quality.yml:19]: Nice didn't know that. better than using a secret indeed!

**Pull request merged**